src/svelte/FormSteps.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  export let labels: Array<string> = [];
+  export let completed: number = 0;
+  export let completedText = 'Ferdig';
+  export let currentText = 'Nåværende';
+  export let ariaValueText = `Steg ${completed + 1} av ${labels.length}`;
```

Comment:
It reads out which step is completed, when you would think it reads out which step you are currently on.
```suggestion
  export let ariaValueText = `Steg ${completed + 2} av ${labels.length}`;
```

Reply:
Does this need to have `export`?

Reply:
I think it should be so it can be translated